crt_portal/cts_forms/templates/forms/base.html

```diff
@@ -99,8 +104,13 @@
     </div>
     {% endblock %}
   </body>
+  {% compress js %}
   <script src="{% static 'js/url_params_polyfill.js' %}"></script>
   <script src="{% static 'js/uswds.min.js' %}"></script>
   <script src="{% static 'js/focus_alert.js' %}"></script>
   {% block page_js %}{% endblock %}
+  {% endcompress %}
+
+  {# do not compress #}
+  {% block touchpoint_js %}{% endblock %}
```

Comment:
```suggestion
  {% block touchpoints_js %}{% endblock %}
```

Reply:
Oh gosh, egg on my face. Thanks for the catch!!